src/main/resources/migrations/00013_add_fields_for_refunds.sql

```diff
@@ -0,0 +1,12 @@
+--liquibase formatted sql
+
+--changeset uk.gov.pay:add_refund_related_fields_to_transaction_table
+
+CREATE type transaction_type_t as enum ('PAYMENT', 'REFUND');
```

Comment:
do you name types with suffix `_t`? Is it needed?

Reply:
- Updated to `00014..`
- We don't have precedent for types yet. Similar to `idx` we have added `_t` for type

Reply:
I just wonder whether it makes sense to suffix types. Especially that the name itself mentions `type`.
`idx` is more descriptive than `t` (which could even mean table, right?)
Feels a bit like Hungarian notation. That's why I wonder whether it's a good practice.
I can approve, but I still think it should be reconsidered.

Reply:
The `type` in the name is not to do with an type - it's the transaction *type* type - if that makes sense? transaction_type exists outside the fact that this is the type for that column